specification/attestation/data-plane/Microsoft.Attestation/stable/2018-09-01-preview/attestation.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "AttestationClient",
     "description": "Describes the interface for the per-tenant enclave service.",
-    "version": "2018-09-01"
+    "version": "2018-09-01-preview"
```

Comment:
Can you please follow the location convention as the other services? Meaning, there's a preview folder that contains your preview specs, once you get to GA you can place your spec under stable version.

Reply:
After chatting offline, this is a "long term stable version" so is ok to leave it in stable folder.